deps.ffmpeg/40-aom.zsh

```diff
@@ -104,6 +114,13 @@ install() {
   if (( _loglevel > 1 )) args+=(--verbose)
 
   cd ${dir}
+
+  if [[ ${target} == macos-universal ]] {
+    pushd build_universal
+    sed -i -E -e 's/build_x86_64/build_universal/g' cmake_install.cmake
```

Comment:
What's the rationale on this?

Reply:
Because `aom` cannot be built as a universal library, the installed files from the `x86_64` and `arm64` build directories are combined into a universal build after the fact.

The cmake install script inherited from the `x86_64` will still retain references to that architecture however, so it would install files from there instead of the manually created `build_universal` directory.

This patch ensures that the install script will copy the "fat" binaries created by the mixup.

Reply:
Got it. A little unfortunate, but oh well.